### ISSUE TITLE
In-built stats functions

### ISSUE THREAD

**Issue by vijayantajain** [vijayantajain/Argus#8]
@ppradiplamsal  - I just realized that there are probably in-built functions in the JavaScript library. We are re-inventing the week with average, max, and min! We should probably look into it.

I will look into it tomorrow.

**Comment by ppradiplamsal** [comment]
Yeah D3 has all of the basic statistical functions. I was wondering what you were trying to do. We should use them.

**Comment by ppradiplamsal** [comment]
Yeah let's get through with the visualization since it is an absolute necessity. We can create the CLI app later I guess.

**Comment by vijayantajain** [comment]
Actually it is not completely necessary. I checked with him. He said if just do a simple cli app, that would be fine as well. But I agree with your decision.
